src/zmap.1

```diff
@@ -212,8 +219,8 @@ Minimum hitrate that scan can hit before scan is aborted
 Comma\-separated list of cores to pin to
 .
 .TP
-\fB\-\-ignore\-invalid\-hosts\fR
-Ignore invalid hosts in whitelist/blacklist file
+\fB\-\-ignore\-blacklist\-errors\fR
+Ignore invalid entries in whitelist/blacklist file
```

Comment:
It might be worth it to be explicit here. IIRC, this doesn't ignore malformed entries, it just allows names that can't be resolved.